internal/connect/command_offset_status.go

```diff
@@ -0,0 +1,23 @@
+package connect
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+)
+
+type offsetStatusCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
```

Comment:
This is not needed, since this subcommand shares the same annotation as its parent.
```suggestion
```